go/vt/vtgate/vschemaacl/vschemaacl.go

```diff
@@ -38,17 +35,17 @@ var (
 	acl map[string]struct{}
 )
 
-func registerSchemaACLFlags(fs *pflag.FlagSet) {
+// RegisterSchemaACLFlags installs log flags on the given FlagSet.
+//
+// `go/cmd/*` entrypoints should either use servenv.ParseFlags(WithArgs)? which
+// calls this function, or call this function directly before parsing
+// command-line arguments.
+func RegisterSchemaACLFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&AuthorizedDDLUsers, "vschema_ddl_authorized_users", AuthorizedDDLUsers, "List of users authorized to execute vschema ddl operations, or '%' to allow all users.")
 }
 
 // Init parses the users option and sets allowAll / acl accordingly
 func Init() {
-	// register flag
-	for _, cmd := range []string{"vtadmin", "vtcombo", "vtgate", "vtgateclienttest"} {
-		servenv.OnParseFor(cmd, registerSchemaACLFlags)
-	}
-
```

Comment:
why did you remove this?

Reply:
So I made mistake. I have reverted back my change. Vschemaacl has Init() method which I mixed up with init(). I put registration in Init() and expected the flags to be available for binaries but it was not happening thats why I moved them under each binary endpoint. But now I realized it was because of Init() is not "init()" :D